packages/benchmark/src/commands/list.ts

```diff
@@ -0,0 +1,21 @@
+import { Command } from '@oclif/core';
+import { TestScenarioLoader } from '@/testScenario/testScenarioLoader';
+import { loadConfig } from '@/config/config';
+
+export default class ListCommand extends Command {
+	static description = 'List all available test scenarios';
+
+	async run() {
+		const config = loadConfig();
+		const testScenarioLoader = new TestScenarioLoader();
+
+		const allScenarios = testScenarioLoader.loadAllTestScenarios(config.get('testScenariosPath'));
+
+		console.log('Available test scenarios:');
```

Comment:
Why not use `packges/cli/src/Logger.ts` instead? Same elsewhere in this package.

Reply:
I don't want this package to depend on `cli`